src/chapter2.adoc

```diff
@@ -104,23 +113,34 @@ The table format is described in <<RQSC_TABLE>>.
 |===
 |Field {set:cellbgcolor:green}|Byte Length|Byte Offset|Description
 4+<|{set:cellbgcolor:!} *Header*
-<|- Resource Type               |1  |0 <a|
+<|- Resource Type               |1  |0  <a|
                                             - 0 - Cache
                                             - 1 - Memory
                                             - 2 - 0x7F - Reserved
                                             - 0x80 - 0xFF - Vendor Specific
-<|- Length                      |1  |1 <a| Length of this specific Resource Structure.
+<|- Reserved                    |1  |1  <a|Reserved. Must be 0
+<|- Length                      |2  |2  <a| Length of this specific Resource Structure.
                                             Length includes the Resource Specific
-                                            Data bytes as well. If length is set to 16,
+                                            Data bytes as well. If length is set to 20,
                                             then, it indicates there is no resource
                                             specific data available for this structure.
-<|- Resource Flags              |2  |2 <a|Resource Type Specific flags. Refer to
+<|- Resource Flags              |2  |4  <a|Resource Type Specific flags. Refer to
                                             <<RESOURCE_FLAGS_TABLE>> for details.
-<|- Resource ID 1               |8  |4 <a|Depends on the Resource Type field. Refer
+<|- Reserved                    |1  |6  <a|Reserved. Must be 0
+<|- Resource ID Type            |1  |7  <a|Type of the Resource ID described by
+                                            Resource ID 1 and Resource ID 2 fields.
+
+                                            - 0 - Cache
+                                            - 1 - Memory Affinity Structure
```

Comment:
I think keeping it just as "memory" here as the resource ID type is better. While explaining for this Memory resource ID type, the description will point to Memory Affinity Structure anyway.